ultimate-kata-vue/public/index.html

```diff
@@ -6,7 +6,42 @@
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
     <title><%= htmlWebpackPlugin.options.title %></title>
+    <style>
+      header { 
+        box-sizing: border-box;
+        display: flex;
+        flex-direction: row;
+        padding: 0 15px;
+        border-bottom-style: solid;
+        border-bottom-color: black;
+        border-bottom-width: 1px;
+        }
+      .logo > img, .logo span { 
+        display: inline-block;
+        height: 80px;
+        vertical-align: bottom;
+        }
+      .header-items {
+        align-self: center;
+        width: 50%;
+        margin-left: auto;
+      }
+      .header-items p {
+        margin-top: 0;
+        text-align: right;
+      }
+    </style>
   </head>
+  <header>
```

Comment:
How comes you have added a header here and not created a component for the header?

Reply:
I figured if the header is unchanging and if someone running a web browser on a toaster tries to view the site, at least the header with Logo and basic menu can show up.
However if you suggest I have the header or part of the header be a component then I can get down with that.

Reply:
It is a good reason... The only counter-argument is if in the future the header gets complicated. With menus or differences between mobile and desktop. It would be much easier in a component. 

Regarding the Toaster issue. If you wanted to make sure the website was viewable you would check the browser at load and probably serve them a static website instead.  

However  it is completely fine for now. I can just see it becoming complicated in the future